test/tools/libtesteth/TestHelper.cpp

```diff
@@ -534,6 +535,33 @@ void executeTests(const string& _name, const string& _testPathAppendix, const st
 	}
 }
 
+void postParseJson(json_spirit::mValue& _obj)
+{
+	if(_obj.type() == json_spirit::obj_type)
+	{
+		for (auto& i: _obj.get_obj())
+		{
+			if (i.first.substr(0, 2) == "//")
+			{
+				_obj.get_obj().erase(_obj.get_obj().find(i.first));
```

Comment:
Do not erase an element of `_obj.get_obj()` while going through `_obj.get_obj()`.

This caused the problem https://github.com/ethereum/cpp-ethereum/issues/4316